fem/eltrans.hpp

```diff
@@ -207,7 +211,12 @@ class InverseElementTransformation
          Use the point returned by FindClosestRefPoint() from a reference-space
          grid of type and size controlled by SetInitGuessPointsType() and
          SetInitGuessRelOrder(), respectively. */
-      GivenPoint = 3 ///< Use a specific point, set with SetInitialGuess().
+      GivenPoint = 3, ///< Use a specific point, set with SetInitialGuess().
+      EdgeScan = 4, /**< Performs full solves on multiple points along the edges
+          of the element. It is recommended that SetInitGuessRelOrder() is chosen
```

Comment:
Seems like it is a hard requirement to `SetInitGuessPointsType()` when using the `EdgeScan` (unlike the other options) on device. It might be worth switching `guess_points_type` to `ClosedUniform` if it is `Invalid` when `init_guess_type` is set to `EdgeScan`.